tests/test_sse.py

```diff
@@ -390,7 +390,33 @@ async def func(request):
     await runner.cleanup()
 
 
-@pytest.mark.asyncio
+async def test_connection_is_not_alive(loop, unused_tcp_port, session):
+    async def func(request):
+        # within context manager first preparation is already done
+        async with sse_response(request) as resp:  # type: EventSourceResponse
+            resp.ping_interval = 1
+
+            # we should sleep to switch asyncio Task
+            # and let connection to be closed
+            await asyncio.sleep(0.1)
+
+            # this call should be cancelled, cause connection is closed
+            await resp.prepare(request)
+
+            return resp  # pragma: no cover
```

Comment:
I feel like this is the main part of the test? But, it doesn't assert anything.
What exactly are we testing? If it's that the task is cancelled, that's handled by aiohttp, I don't think it's related to aiohttp-sse.

Reply:
No, wait, aiohttp doesn't cancel tasks by default anymore. Ignore that.

But, I think we need to atleast do a `with pytest.raises(asyncio.CancelledError)` or similiar to actually assert something.

Reply:
Added raises check